File: android/src/main/java/me/carda/awesome_notifications/awesome_notifications_android_core/utils/BooleanUtils.java
package me.carda.awesome_notifications.awesome_notifications_android_core.utils;

public class BooleanUtils {
    public static boolean getValue(Boolean booleanObject){
        return booleanObject != null && booleanObject;
    }
    public static boolean getValueOrDefault(Boolean booleanObject, Boolean defaultValue){
        return booleanObject == null ? getValue(defaultValue) : getValue(booleanObject);
    }
}

File: ios/Classes/lib/core/utils/BooleanUtils.swift
//
//  BooleanUtils.swift
//  awesome_notifications
//
//  Created by Rafael Setragni on 05/09/20.
//

import Foundation

class BooleanUtils {
    
    public static func getValue(value: Any?, defaultValue: Bool?) -> Bool {
        return value as! Bool? ?? defaultValue ?? false
    }
    
}
